client/src/index.js

```diff
@@ -1,10 +1,16 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from "./App";
+import * as serviceWorker from "./serviceWorker";
+import { UserProvider } from "./context/user";
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(
+  <UserProvider>
+    <App />
+  </UserProvider>,
```

Comment:
It would be better to put the UserProvider as the top level element inside of App, rather than wrap App with it

Reply:
Thanks!

Reply:
I ended up wrapping App with UserProvider in index.js because I was not able to access UserContext in App.js